src/Fade.js

```diff
@@ -2,87 +2,77 @@ import React from 'react';
 import Transition from './Transition';
 
 class Fade extends React.Component {
-
-  constructor(props, context){
-    super(props, context);
-  }
```

Comment:
if you only call `super` in the constructor with the same arguments its the same thing as not having it. Classes will essentially create that constructor automatically for you.